From a1c974dcae475c5f949ad9853cb7ffe452d8e0f6 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 18 Jan 2019 23:24:15 +0300
Subject: [PATCH 843/909] ARM64: dts: renesas: ulcb: Make AK4613 sound device
 name consistent

commit 3d10977e9e9811efe110d90b725f3132f80112ae from
https://github.com/CogentEmbedded/meta-rcar.git

This makes AK4613 sound device name consistent throughout all
ULCB boards. This helps to avoid naming issues when pulseaudio
fails to start because the device has different names on
infotainment and starter boards. For example, "ak4613"
on Kingfisher vs "rsnd-sound" on Starter Kit.
Pulseaudio is expecting "ak4613" name.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/ulcb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/ulcb.dtsi b/arch/arm64/boot/dts/renesas/ulcb.dtsi
index b60fd45f0a17..7e546d340364 100644
--- a/arch/arm64/boot/dts/renesas/ulcb.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb.dtsi
@@ -97,7 +97,7 @@
 	sound_card: sound {
 		compatible = "audio-graph-card";
 
-		label = "rcar-sound";
+		label = "ak4613";
 
 		dais = <&rsnd_port0>;
 	};
-- 
2.17.1

